bootstrap.sh

```diff
@@ -1 +1,19 @@
 #!/bin/bash
+
+# crating database
+kubectl apply -f ./.infrastructure/mysql/namespace.yml
+kubectl apply -f ./.infrastructure/mysql/configMap.yml
+kubectl apply -f ./.infrastructure/mysql/secret.yml
+kubectl apply -f ./.infrastructure/mysql/service.yml
+kubectl apply -f ./.infrastructure/mysql/statefulSet.yml
+
+#creating app
+kubectl apply -f ./.infrastructure/namespace.yml
+kubectl apply -f ./.infrastructure/confgiMap.yml
```

Comment:
There is a typo in the file path: 'confgiMap.yml' should be 'configMap.yml'. Ensure the file name is correct.